apps/web/src/services/cost-explorer/composables/use-budget-create-mutation.ts

```diff
@@ -20,7 +20,10 @@ export const useBudgetCreateMutation = (options: UseBudgetCreateMutationOptions)
     return useMutation({
         mutationFn: budgetAPI.create,
         onSuccess: async (data) => {
-            if (onSuccess) await onSuccess(data);
+            if (onSuccess) {
+                await onSuccess(data);
+                invalidateBudgetListQuery();
+            }
```

Comment:
Shouldn't invalidation be performed without callback?

```suggestion
invalidateBudgetListQuery()
if (onSuccess) await onSuccess(data);
```